src/snek/terminal.rs

```diff
@@ -106,3 +167,12 @@ impl GameDimensions {
     self.width <= rhs.width && self.height <= rhs.height
   }
 }
+
+impl GameCoordinate {
+  fn adjusted(self, TerminalOffset { x, y }: TerminalOffset) -> Self {
+    GameCoordinate {
+      x: self.x + x,
+      y: self.y + y,
+    }
+  }
+}
```

Comment:
don't think we need this since we ended up defining the `adjust` closure

Reply:
good call

Reply:
ironically, this (and other comments) are slipping by because of that allow(dead_code) lol